src/main/scala/com/hortonworks/spark/sql/hive/llap/LlapRelation.scala

```diff
@@ -133,7 +138,7 @@ case class LlapRelation(
   }
 
   private def getQueryString(requiredColumns: Array[String], filters: Array[Filter]): String = {
-    var selectCols = "*"
+    var selectCols = "1"
```

Comment:
Hi, @jdere .
Thanks. But, what I meant is that we need to execute `SELECT COUNT(1)` instead of `SELECT 1`.
When we get the total number of rows, we can make iterator with that number of empty rows.
That will reduce the traffic greatly. How do you think about that?

cc @merlintang .